aspnetcore/fundamentals/servers/kestrel/http3.md

```diff
@@ -20,29 +20,19 @@ uid: fundamentals/servers/kestrel/http3
 > [!IMPORTANT]
 > Apps configured to take advantage of HTTP/3 should be designed to also support HTTP/1.1 and HTTP/2.
 
-## HTTP/3 requirements
-
-HTTP/3 has different requirements depending on the operating system. If the platform that Kestrel is running on doesn't have all the requirements for HTTP/3, then it's disabled, and Kestrel will fall back to other HTTP protocols.
-
-### Windows
-
-* Windows 11 Build 22000 or later OR Windows Server 2022.
-* TLS 1.3 or later connection.
-
-### Linux
-
-* `libmsquic` package installed.
```

Comment:
@CarnaViire Shouldn't this stay in .NET 8 and lower versions of this document? Ditto for all the other information you deleted. Tell me what needs to stay.

Reply:
None of the requirements needs to stay for any version. The new paragraph links to [`System.Net.Quic`](https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/quic/quic-overview#platform-dependencies) requirements that cover the necessary info about which MsQuic version is compatible with what .NET version

Reply:
No changes for .NET 9+? If so, I'll remove my .NET 8 version and have one file for .NET 8+